conf/broker.conf

```diff
@@ -314,28 +311,19 @@ loadBalancerHostUsageCheckIntervalMinutes=1
 
 # Load shedding interval. Broker periodically checks whether some traffic should be offload from
 # some over-loaded broker to other under-loaded brokers
-loadBalancerSheddingIntervalMinutes=30
+loadBalancerSheddingIntervalMinutes=10
 
 # Prevent the same topics to be shed and moved to other broker more that once within this timeframe
-loadBalancerSheddingGracePeriodMinutes=30
-
-# Usage threshold to determine a broker as under-loaded
-loadBalancerBrokerUnderloadedThresholdPercentage=1
+loadBalancerSheddingGracePeriodMinutes=20
```

Comment:
`loadBalancerBrokerUnderloadedThresholdPercentage` is missing in new file

Reply:
`loadBalancerBrokerUnderloadedThresholdPercentage` is only used by `SimpleLoadManagerImpl` so, I have deprecated all these configs as we are changing load-manager as `ModularLoadManager` and also removed these configs from `broker.conf` and `standalone.conf`

Reply:
or we can still keep in broker.conf ? I think I will bring back all those removed configs and  we already have comment that this config only used by SimpleLoadManagerImpl.

Reply:
Oh ok, if the `ModularLoadManager` doesn't use these, we should retire them from the conf files. Otherwise it might seem that they're applicable